airflow/jobs.py

```diff
@@ -1790,6 +1790,58 @@ class BackfillJob(BaseJob):
         'polymorphic_identity': 'BackfillJob'
     }
 
+    class _TiStatus(object):
```

Comment:
sounds like it is task status? TI is defined by task (task_id + dag_id) and execution_date